<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogManageRoutes</class>
 <widget class="QDialog" name="DialogManageRoutes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Routes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QComboBox" name="sniffing_mode">
         <item>
          <property name="text">
           <string>Disable</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sniffing to DNS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sniffing to destination</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Sniffing Mode</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Outbound Domain Strategy</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Remote DNS</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="outbound_domain_strategy">
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string notr="true">AsIs</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">UseIP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">UseIPv4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">UseIPv6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">PreferIPv4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">PreferIPv6</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="dns_remote"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="dns_direct"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Direct DNS</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,1,0,0">
         <item>
          <widget class="QCheckBox" name="dns_routing">
           <property name="text">
            <string>Enable DNS Routing</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Domain Strategy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="domainStrategyCombo">
           <property name="editable">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string notr="true">AsIs</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">IPIfNonMatch</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">IPOnDemand</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Matcher</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="domainMatcherCombo">
           <item>
            <property name="text">
             <string notr="true">Original</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Minimal Perfect Hash Matcher</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,1,1" columnstretch="0,1,1,1">
         <item row="0" column="3">
          <widget class="QLabel" name="label_82">
           <property name="text">
            <string>Block</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <layout class="QGridLayout" name="blockTxtLayout"/>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_80">
           <property name="text">
            <string>Direct</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Domain</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QGridLayout" name="directTxtLayout"/>
         </item>
         <item row="1" column="2">
          <layout class="QGridLayout" name="proxyIPLayout"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_81">
           <property name="text">
            <string>Proxy</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <layout class="QGridLayout" name="proxyTxtLayout"/>
         </item>
         <item row="1" column="3">
          <layout class="QGridLayout" name="blockIPLayout"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>IP</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QGridLayout" name="directIPLayout"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="preset">
         <property name="text">
          <string>Preset</string>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="custom_route_edit">
         <property name="text">
          <string>Custom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sniffing_mode</tabstop>
  <tabstop>outbound_domain_strategy</tabstop>
  <tabstop>dns_remote</tabstop>
  <tabstop>dns_direct</tabstop>
  <tabstop>dns_routing</tabstop>
  <tabstop>domainStrategyCombo</tabstop>
  <tabstop>domainMatcherCombo</tabstop>
  <tabstop>preset</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>DialogManageRoutes</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogManageRoutes</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
